app/controllers/to_dos_controller.rb

```diff
@@ -0,0 +1,41 @@
+class ToDosController < ApplicationController
+  before_action :set_to_do, only: [:update, :destroy]
+
+  def index
+    @to_dos = ToDo.all
+    render json: @to_dos
+  end
+
+  def create
+    @to_do = ToDo.new(to_do_params)
+    if @to_do.save
+      render json: @to_do
+    else
+      render json: @to_do.errors
+    end
+  end
+
+  def update
+    @to_do.update(to_do_params)
```

Comment:
Small point: `update` here will perform the save, so believe it or not you're saving the record twice here. You can just do `if @to_do.update(to_do_params)` and it'll do the same thing slightly more efficiently.